client/ayon_core/plugins/publish/extract_color_transcode.py

```diff
@@ -154,12 +153,15 @@ def process(self, instance):
 
                 files_to_convert = self._translate_to_sequence(
                     files_to_convert)
+                self.log.debug("Files to convert: {}".format(files_to_convert))
                 for file_name in files_to_convert:
+                    self.log.debug("Transcoding file: `{}`".format(file_name))
                     input_path = os.path.join(original_staging_dir,
                                               file_name)
                     output_path = self._get_output_file_path(input_path,
                                                              new_staging_dir,
                                                              output_extension)
+                    self.log.debug("Ynput path: `{}`".format(input_path))
```

Comment:
This seems very verbose with all these added logs. Intentional? or just for debugging?

```suggestion
                    self.log.debug("Input path: `{}`".format(input_path))
```

Reply:
Huh, and I was assuming this was removed, thanks!